operations/package_repository_management/Nexus_Export_and_Restore.md

```diff
@@ -0,0 +1,146 @@
+# Nexus Export and Restore process
+
+The current process for ensuring the saftey of the nexus data is a one time, space intesive, manual process and is only recommeded to be done while nexus is in a known good state. The process is needed to be done if a second copy of the nexus data is needed.
```

Comment:
maybe I'm reading the commits in the wrong order. If you've already fixed these, lmk.